xiilib/flask/charm.py

```diff
@@ -1,230 +1,113 @@
 #!/usr/bin/env python3
+# pylint: disable=duplicate-code
```

Comment:
Would you be able to provide a context for this disable please? Perhaps scoping it to the disabled code would be better since it will not re-enable itself.

Reply:
This is originally for the Django `charm.py` which shares a lot of initialization sequence, but I forgot to remove this, I will remove this in the Django pull request, thanks!